package.json

```diff
@@ -53,7 +53,7 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^2.7.1",
     "ts-jest": "^29.0.3",
-    "typescript": "^4.8.4",
+    "typescript": "^4.9.5",
```

Comment:
Needed to upgrade typescript to reproduce the downstream build issue, but I cannot use the latest because of this issue: https://github.com/webdriverio/webdriverio/issues/10280

Need to upgrade to webdriverio v8 before upgrading TS